extension/content_script.js

```diff
@@ -281,6 +284,15 @@ function highlight(sentenceId) {
 	highlightedSentenceId = sentenceId;
 };
 
+function persistentHighlight() {
```

Comment:
1. Please leave a comment:
// Toggles the persistent highlight on the currently tracked sentence

2. I think this will throw an error when tracker is not pointing at anything (I forgot if tracker initially points to something).
To be safe, u can do:
if (!tracker.isTracking()) {
  return;}

Reply:
(It does initially point to something; will add all the same)